app/controllers/contacts_controller.rb

```diff
@@ -6,14 +6,9 @@ def search
     assert_edition_state(@edition, &:editable?)
     assert_edition_access(@edition, current_user)
     @contacts_by_organisation = ContactsService.new.all_by_organisation
+    render layout: rendering_context
   rescue GdsApi::BaseError => e
     GovukError.notify(e)
     render "search_api_down", status: :service_unavailable
```

Comment:
Sorry for not spotting this the first time round: this doesn't actually get rendered in the modal as it's not a 2xx response code. Noting that the response code is somewhat inaccurate (our app server is available), I wonder if we should actually use the default 200 status and make it work with the modal - it's the only use of the modal which relies on an external API being up.

Reply:
Updated to use the API down message for both the modal and no-JS experience.